pepys_import/resolvers/command_line_resolver.py

```diff
@@ -450,17 +449,25 @@ def fuzzy_search_platform(
         completer = list()
         platforms = data_store.session.query(data_store.db_classes.Platform).all()
         for platform in platforms:
-            completer.append(
-                f"{platform.name} / {platform.identifier} / {platform.nationality_name}"
-            )
+            p = f"{platform.name} / "
```

Comment:
@BarisSari - would this code fail if the users used a '/' in a field?

Wouldn't it be more robust to build up a list?

Reply:
Yes, it might fail because we split the selected option by "/" character. I'm changing the strategy. You're right that it would be more robust if we had a list

Reply:
Actually, I checked the code again and it doesn't fail when user used a "/" in a field. Because we split by " / ", not only "/" characters. So, that's why we didn't have any problems until now. Do you still think that we should change it to list?

Reply:
Yes, I still think we should switch to a list, please.  It looks much more robust.

Reply:
okay